vars/writeBuildInfo.groovy

```diff
@@ -4,12 +4,12 @@ import groovy.json.JsonBuilder
 import groovy.json.JsonSlurperClassic
 import org.feedhenry.Utils
 
-def call(name, versionTxt) {
+def call(name, versionText, parseVersionText = true) {
```

Comment:
You could consider adding types here, since you're changing it anyway. Not required though!